src/dropdown/Dropdown.js

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
+import { props, t, skinnable } from '../utils';
 import Select from 'react-select';
-import omit from 'lodash/omit';
+import find from 'lodash/find';
 import cx from 'classnames';
-import { props, t } from '../utils';
 import { warn } from '../utils/log';
 
-const themes = {
-  semantic: 'semantic-theme'
-};
-
 const PropTypes = {
-  children: t.maybe(t.ReactNode),
-  theme: t.maybe(t.enums.of(['semantic'])),
+  value: t.maybe(t.union([t.Number, t.String, t.Object])),
   valueLink: t.maybe(t.struct({
     value: t.maybe(t.String),
```

Comment:
as you improved the evaluation for `value`, can you improve this too?
(`value: t.maybe(t.union([t.Number, t.String, t.Object])),`)